examples/zephyr/common/Kconfig

```diff
@@ -67,7 +67,6 @@ config GOLIOTH_SAMPLE_DHCP_BIND
 config GOLIOTH_SAMPLE_FW_VERSION
 	string "DFU Firmware Version"
 	default "0.0.0+0"
-	depends on BOOTLOADER_MCUBOOT
```

Comment:
Please add `examples: zephyr: ` in commit title for better navigation across commits.

Reply:
@mniestroj done!